core/drivers/crypto/hisilicon/include/hpre_main.h

```diff
@@ -6,6 +6,7 @@
 #define _HPRE_MAIN_H
 
 #include <initcall.h>
+#include "rng_support.h"
```

Comment:
Please keep the include list sorted and use `<>` where possible.

Reply:
solved